README.md

```diff
@@ -46,31 +46,17 @@ run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gath
 ### Building locally for testing
 You need to be autheticated against the OpenShift registry CI, registry.ci.openshift.org in order to be able to pull the images in the `Dockerfile`.
 
-Before proceeding make sure you have `oc` (OpenShift cli) and `jq` (optional) installed.
+Before proceeding make sure you have `oc` (OpenShift cli) installed.
 
 Follow these steps to authenticate:
 
 - Navigate to the OpenShift console for `app.ci` cluster located [here](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/).
 - Login using SSO
 - Go to your profile name in the top right and select "Copy login command".
```

Comment:
instead of copy the login command it would be better to mention copy the token from the login command.

Reply:
Ah! The menu is actually labeled "copy login command" hence the statement.

Regards

Reply:
I think there is a slight misunderstanding, the drop down menu is called "copy login command". 

Hence it is written like that. The next step tells user to copy the displayed token. 

TLDR; You cannot get to token screen without clicking on "copy login command" from the overflow menu. 

Hope this helps clarify.

Reply:
ohh ops, didn't notice that. cool LGTM then.